.env.example

```diff
@@ -4,3 +4,10 @@ GITHUB_ANALYZER_PASSWORD=postgres
 
 # Must be set as the webhook secret on your github app
 GITHUB_ANALYZER_WEBHOOK_SECRET=fake-secret
+
+# Token obtained from https://codeclimate.com/profile/tokens
+CODE_CLIMATE_API_TOKEN:
```

Comment:
Isn't it necessary for the tests?

Reply:
No, the API calls are partially mocked in the tests.